src/components/TheSummary.vue

```diff
@@ -143,16 +143,16 @@ export default {
       const summary = new Summary()
       let sports: Set<string> = new Set()
 
-      let gapSeconds: number = 0
       for (let i = 0; i < this.selectedSessions.length; i++) {
+        let gapSeconds: number = 0
```

Comment:
so now gapSeconds always between 0 or (currentFirstTimestamp - prevLastTimestamp) / 1000? cmiiw

Reply:
yes, for example I ride for a coffee and recording it, when I got into coffee shop I stop the recording (1st session), then I move to the next destination and I start recording again (2nd session), there is a gap between first session and the second session, and it should be added to `TotalElapsedTime`.

Reply:
So the gap calculation is only between A-B sessions and B-C sessions and so on.